a10_octavia/tests/unit/controller/worker/tasks/test_handler_server.py

```diff
@@ -0,0 +1,48 @@
+#    Copyright 2020, A10 Networks
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import acos_client
+import mock
+from unittest.mock import patch
+
+from octavia.tests.unit import base
+from octavia.common import data_models as o_data_models
+from octavia.tests.common import constants as t_constants
+
+from a10_octavia.controller.worker.tasks.handler_server import MemberCreate
+from a10_octavia.common.data_models import VThunder
+from a10_octavia.tests.common import a10constants
+
+VTHUNDER = VThunder()
+POOL = o_data_models.Pool(id=a10constants.MOCK_POOL_ID)
+MEMBER = o_data_models.Member(
+    id=a10constants.MOCK_MEMBER_ID, protocol_port=t_constants.MOCK_PORT_ID)
+
+
+class TestMemberCreate(base.TestCase):
+
+    def setUp(self):
+        patcher = patch(
```

Comment:
Instead we could do

```python
mock_patch = patcher.start()
self.client_mock = mock.Mock()
mock_patch.return_value = self.client_mock
```

Then later on we just have to check ` self.client_mock.slb.server.delete.assert_called_with`